README.md

```diff
@@ -1,4 +1,4 @@
-![SQLECTRON Logo](https://www.dropbox.com/s/038vmsywakz2hug/SQLECTRON-400px.png?raw=1)
+![SQLECTRON Logo](https://www.dropbox.com/s/dt06tif4xo1khbf/SQLECTRON-400px.png?raw=1)
```

Comment:
It would't be better to place into the app the image, i can imagine the scenario where we are without internet connection and it might not load the logo image